packages/suite/src/components/wallet/AccountMode/index.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { AccountWatchOnlyMode } from '@wallet-reducers/selectedAccountReducer';
 
 import AuthConfirmFailed from './AuthConfirmFailed';
-import LoadingOtherAccounts from './LoadingOtherAccounts';
+// import LoadingOtherAccounts from './LoadingOtherAccounts';
```

Comment:
my father always taught me: "A good review can't go without a totaly useles nitpick". 

So, is there any need for this comment?

Reply:
forgot to delete that components!